### DISCUSSION TITLE
음성 채팅 기능 관련

### DISCUSSION THREAD

**Discussion by strangestraw** [naverz/zepeto-studio-kor#211] [5. Q&A]
안녕하세요.
유니티로 생성한 월드에서 빌드잇의 음성 채팅 기능을 사용할 수 있나요?
가이드 부탁드립니다.
감사합니다.

**Comment by moondory77-naverz** [comment]
안녕하세요. 
현재 Unity를 통한 월드개발에서, 음성 채팅 기능은 제공드리고 있지 않습니다.
해당 기능의 추가를 내부 검토중에 있으며
추후 업데이트 시, 안내 드리도록 하겠습니다. 

감사합니다.

**Reply by strangestraw** [reply]
감사합니다~

**Comment by strangestraw** [comment]
그럼 아래의 경우는 가능할까요?

1. 제페토 멀티플레이 기능으로 음성 채팅 구현
2. 외부 솔루션 도입으로 음성 채팅 구현

가이드 부탁드립니다.

감사합니다.

**Reply by moondory77-naverz** [reply]
안녕하세요.
외부솔루션을 도입하여, 제페토 멀티플레이에서 음성채팅을 직접구현하기 위해서는
외부솔루션의 Unity SDK(C#)를 사용하는 방식이 될 듯 한데요.

안타깝지만 제페토 월드 제작의 특성 상, 패키지에 C# 스크립트를 혼용하는 방식은 불가합니다. ㅠ [참고](https://github.com/naverz/zepeto-studio-kor/discussions/138#discussioncomment-1868185)

**Reply by cheolhui-lee** [reply]
제페토 멀티플레이 기능으로 오디오 데이터를 전송한다면 데이터의 전달 관점에서 불가능할 것 같지는 않지만,
시스템의 설계 목표가 애당초 달라서 오디오 데이터를 안정적으로 충분히 잘 전달 할 수 있을 지에 대해서는 많은 의문이 있습니다.
대략 상상해보면 전달 받은 데이터를 플레이 하면 빈번하게 소리가 깨질 것 같네요.
